job-app/src/App.js

```diff
@@ -87,7 +103,7 @@ class App extends React.Component {
                             </div>
                         </Container>
                         <div className="options">
-                            <Button variant="primary" className="btn-primary" onClick={() => { this.setState({popupOpen: true}); }}>+ New Application</Button> {' '}
+                            <Button variant="primary" className="btn-primary" onClick={() => { this.setState({popupOpenAdd: true}); }}>+ New Application</Button> {' '}
```

Comment:
same thing here

Reply:
oh wait that's something I think I might've done. Ignore this for now

Reply:
Ok can you remove the "className='btn-primary'" part here and the one I commented above? Those aren't necessary, but I had originally added those but realized I didn't import the react-bootstrap css file correctly